packages/components/src/utils/hooks.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect, useRef } from '@wordpress/element';
+
+/**
+ * An enhanced useState hook that stores and updates an internal state based on
+ * an incoming state value.
+ *
+ * This is useful for input/control based components.
+ *
+ * @param {any} initialState
+ *
+ * @return {Array<any, Function>} Controlled [state, setState] values.
+ */
+export function useControlledState( initialState ) {
```

Comment:
This doesn't seem used right?

Reply:
Ah! You are correct. It was previously used before we implemented `Composite` from Reakit.
I'll remove it now. Thank you 🙏